PIPELINES.md

```diff
@@ -0,0 +1,46 @@
+# Pipelines
+
+Each project should have a `pipeline.sh` script that will be run nightly to
+install and test your code and alert if there are regression failures.
+
+## pipeline.sh
+
+The `pipeline.sh` script:
+
+- is run whenever a Pull Request changes that project
+- is run for all projects each night
+- is run on a private continuous integration server after an initial code
+ review, to reduce the risk of exposing secrets
```

Comment:
let's not reveal too much here. How about `can be triggered manually at any time by ??`